integrations/serverless-ossm-traffic-isolation.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="serverless-ossm-traffic-isolation"]
+= Using {SMProductShortName} to isolate network traffic with {ServerlessProductName}
+:context: serverless-ossm-setup
+
+toc::[]
+
+:FeatureName: Using {SMProductShortName} to isolate network traffic with {ServerlessProductName}
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+{SMProductShortName} can be used to isolate network traffic between tenants on a shared OpenShift cluster using Service Mesh `AuthorizationPolicy` resources. {ServerlessProductShortName} can also leverage this, using several {SMProductShortName} resources. A tenant is a group of one or multiple OpenShift projects that can access each other over the network on a shared OpenShift cluster.
+
+[id="prerequisites_serverless-ossm-traffic-isolation"]
+== Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
```

Comment:
User the the lead-in phrase "You have:" and then remove that from the bullet point text to reduce redundancy.